core/src/main/scala/org/http4s/Uri.scala

```diff
@@ -33,52 +36,53 @@ import scala.reflect.macros.blackbox
   * @param query      optional Query. url-encoded.
   * @param fragment   optional Uri Fragment. url-encoded.
   */
-// TODO fix Location header, add unit tests
 final case class Uri(
     scheme: Option[Uri.Scheme] = None,
     authority: Option[Uri.Authority] = None,
-    path: Uri.Path = "",
+    path: Uri.Path = Uri.Path.empty,
     query: Query = Query.empty,
     fragment: Option[Uri.Fragment] = None)
     extends QueryOps
     with Renderable {
-  import Uri._
 
   /**
     * Adds the path exactly as described. Any path element must be urlencoded ahead of time.
     * @param path the path string to replace
     */
-  def withPath(path: Path): Uri = copy(path = path)
+  @deprecated("Use {withPath(Uri.Path)} instead", "1.0.0-M1")
```

Comment:
since should be checked by someone

Reply:
Does that `{}` syntax do anything in a deprecation warning?  I sometimes backtick them.  Doesn't really matter. Mostly asking for my own curiosity.

Reply:
I`m not sure. For some reason i was thinking scaladoc syntax here. I can change it to backticks if that is what we have used elsewhere